modules/administration/pages/ansible-integration.adoc

```diff
@@ -0,0 +1,52 @@
+[[ansible-integration]]
+= Ansible integration
+
+[NOTE]
+====
+This feature is a techhnological preview.
+====
+
+[NOTE]
+====
+Currently, the supported version of Ansible is 2.9 (LTS) shipped in SUSE Linux Enterprise 15 Client Tools channels. For Control Nodes running operating systems other than SUSE Linux Enterprise, use Ansible shipped by your distribution.
+====
+
+[[at.ansible.overview]]
+== Feature Overview
+
+{productname} enables system administrators to operate their Ansible Control Nodes. Supported features are:
+* introspection of inventory files
+* discovery of playbooks
+* execution of playbooks
+
+
+[[at.ansible.requirements]]
+== Requirements
+
+To use Ansible features, you need to register your Ansible Control Node system as a Salt client in your {productname}. Moreover, you must enable the ``Ansible Control Node`` system type in the [guimenu]``Add-on System Types`` in the menu:System Details[Properties] page of that system.
+
+Enabling this system type ensures that the ``ansible`` package is installed on the system by adding it in the highstate and enables the Ansible features under the menu::System[Ansible] page.
+
+As the next step, configure the paths to your Ansible inventories and playbooks in the menu::System[Ansible][Control Node]. As an invertory path, you can use the typical ansible inventory path [literal]``/etc/ansible/hosts``. As a playbook directory, you can use any directory on the control node, where your playbooks reside (typically directories containing the [literal]``site.yml`` files).
+
+
+[[at.ansible.inventory-introspection]]
+== Inventory introspection
+After defining an inventory path, you can use {productname} to introspect its contents. You can do so in the menu::System[Ansible][Inventories] page. Clicking an inventory path in this page executes the inventory introspection in real-time on the Control Node.
+
+
+[[at.ansible.playbook-discovery]]
+== Playbook discovery
+After defining a playbook directory, you can discover playbooks residing in it via the menu::System[Ansible][Playbooks] page.
+
+As with the inventory introspection, the playbook discovery operations runs in real-time on the Control Node.
+
+
+[[at.ansible.playbook-execution]]
+== Playbook execution
+[NOTE]
+You can schedule a playbook execution from the menu::System[Ansible][Playbooks] page. After selecting the playbook you wish to execute, you can select the inventory file for the execution from the dropdown element (if you don't select any item, the default inventory configured in your Control Node will be used). The dropdown element is populated with the inventories you defined in your Inventory Paths and with inventories that have been locally discovered in your playbook directories (displayed as "Custom Inventory" items in the playbook details).
```

Comment:
You can also type an arbitrary path for the inventory execution instead of selecting from the list.

Reply:
Makes sense. Fixed!